### PR TITLE
DOC: add semicolons to suppress text repr of matplotlib objects in visualization.rst

### PR DESCRIPTION
**Pull request by partev**
add semicolons (";") after the plotting function to suppress unnecessary debug messages similar to:

<matplotlib.axes._subplots.AxesSubplot at 0x7fe278bb5160>

- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Comment by WillAyd** [comment]
Is there a more generic way to do this? I think this is distracting for end users

@jorisvandenbossche

**Comment by MarcoGorelli** [comment]
> I think this is distracting for end users

FWIW, I've seen this quite frequently in tutorials

**Comment by jorisvandenbossche** [comment]
We indeed already do it in several places in visualization.rst itself (see https://pandas.pydata.org/docs/dev/user_guide/visualization.html), so this is making it more consistent.

File: doc/source/user_guide/visualization.rst [hunk 1]
```diff
@@ -286,10 +286,10 @@ subplots:
 
 .. ipython:: python
 
-   plt.figure()
+   plt.figure();
```
Comment (jorisvandenbossche) [inline doc/source/user_guide/visualization.rst:289]: Is this plt.figure() actually needed?

**Comment by jreback** [comment]
@partev can you merge master and we can get this in

**Comment by partev** [comment]
merged

**Comment by jreback** [comment]
thanks @partev 

@jorisvandenbossche has a comment from above if you can have a look

**Pull request merged**